packages/golden-layout/src/utils/index.ts

```diff
@@ -6,3 +6,4 @@ export { default as ReactComponentHandler } from './ReactComponentHandler';
 export * from './ConfigMinifier';
 export { default as BubblingEvent } from './BubblingEvent';
 export { default as EventHub } from './EventHub';
+export * from './EventUtils';
```

Comment:
Do we want to export everything here or just `makeEventFunctions` and the types?

Reply:
I don't see any reason _not_ to export the other functions.

Reply:
My thought was if we expect to use `makeEventFunctions` and not the specific function calls then might as well only export that, but it doesn't hurt anything to export the others